pandas/tests/reshape/concat/test_index.py

```diff
@@ -257,3 +257,10 @@ def test_concat_multiindex_dfs_with_deepcopy(self):
         tm.assert_frame_equal(result_copy, expected)
         result_no_copy = concat(example_dict, names=["testname"])
         tm.assert_frame_equal(result_no_copy, expected)
+
+    def test_concat_multiindex_unique_and_duplicate(self):
+        # GH#44786
+        df = DataFrame({"col": ["a", "b", "c"]}, index=["1", "2", "2"])
+        result = concat([df], keys=["X"])
+        assert result.index.is_unique is False
```

Comment:
Please use assert not … and assert … without is True/False

otherwise lgtm

Reply:
Done, Thanks!